README.md

```diff
@@ -84,6 +84,8 @@ Returns a new focus trap on `element` (one or more "containers" of tabbable node
 
 - **onActivate** {function}: A function that will be called when the focus trap activates.
 - **onDeactivate** {function}: A function that will be called when the focus trap deactivates,
+- **checkCanActivate** {function}: Animated dialogs have a small delay between when `onActivate` is called and when the focus trap is focusable. If checkCanActivate returns false, it will try sending focus again every 5 milliseconds (up to 10 seconds) until it is able to send focus.
+- **onSuccessfulActivation** {function}: This function is only relevant if used in conjunction with `checkCanActivate`. `onActivate` is called before focus is sent to the target. `onSuccessfulActivation` is called after focus is sent to the target.
```

Comment:
Since we have `onActivate()` which is basically like a "pre-activate", how about we rename this one to `onPostActivate()` and state that it occurs after the initial focus node has received focus. It should always be called if specified, independently of `checkCanActivate()`. I don't think the two need to be tied like this.

Reply:
> I don't think the two need to be tied like this.

In the original functionality, `updateTabbableNodes()` is called _before_ `onActivate()`.

`updateTabbableNodes()` is the thing that sends focus to the focus-trap.

If we move `onActivate()` to go before `updateTabbableNodes()` it has a high chance of breaking existing functionality for people.

I made this `if` split to retain backwards compatibility.

We could add an `onBeforeActivate()` setting but that would make it very confusing around when `onActivate()` is actually triggered. 

Maybe `onActivate()` becomes deprecated in favour of `onBeforeActivate()` and `onAfterActivate()`?

Reply:
What I meant was that `updateTabbableNodes()` was the thing causing the error.

I've handled it now in a backwards compatible way.